presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml

```diff
@@ -4,3 +4,4 @@ databases:
   presto:
     host: presto-master
     configured_hdfs_user: hive
+    cli_server_address: http://${databases.presto.host}:8080
```

Comment:
move it to default `tempto-configuration.yaml` in product test resources, so it won't fail when running product tests not only on docker